### PR TITLE
Add Gray effect

### LINKED ISSUE #1: Black/White filter

### PR DESCRIPTION
**Pull request by bradleyboy**
Fixes #1

### CODE REVIEW

File: inyourface/Animator.py [hunk 1]
```diff
@@ -139,9 +139,9 @@ def gif(self):
             outname = self.destdir + self.__class__.name + "/" + self.hash + ".gif"
             self.imdata = urllib.urlopen(self.url).read()
             self.image = Image.open(cStringIO.StringIO(self.imdata))
+            self.secondary_imdata = []
+            self.secondary_image = []
             if (len(self.secondary_urls) > 0):
-                self.secondary_imdata = []
-                self.secondary_image = []
```
Comment (bradleyboy) [inline inyourface/Animator.py:144]: I had to move these out or I got weird errors when doing multi effects

File: inyourface/Animator.py [hunk 2]
```diff
@@ -153,10 +153,10 @@ def gif(self):
             if (self.animated_source):
                 self.total_frames = self.animated_source
                 (frames, durations) = self.generate_frames_from_animation()
-                cmd = "/usr/bin/gifsicle -l0 --colors 255"
+                cmd = "gifsicle -l0 --colors 255"
```
Comment (bradleyboy) [inline inyourface/Animator.py:156]: Can rely on $PATH here? My gifsicle was in `/usr/local/bin`

File: inyourface/effect/Gray.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+from inyourface.Animator import Animator
+
+class EffectAnimator(Animator):
+
+    name = "gray"
+
+    def manipulate_frame(self, frame_image, faces, index):
+        frame_image = frame_image.convert('L')
+        return frame_image
```
Comment (bradleyboy) [inline inyourface/effect/Gray.py:13]: Take it all in. Really savor it.

**Comment by bradleyboy** [comment]
PROOF

fbd793fb5eff74715cb13622fb0bf4f8e5820181

**Comment by yacomink** [comment]
👍 👍 👍

**Pull request merged**